airflow-core/src/airflow/api_fastapi/auth/managers/base_auth_manager.py

```diff
@@ -133,6 +134,15 @@ def get_url_logout(self) -> str | None:
         """
         return None
 
+    def refresh_token(self, refresh_token: str) -> dict | None:
```

Comment:
See the other comment thread for more details but the signature should actually be:
```suggestion
    def refresh_token(self, user: T) -> T | None:
```